docs/pages/desktop-access/rbac.mdx

```diff
@@ -57,14 +57,15 @@ use wildcards (`"*"`) to match all desktop labels.
 
 Windows desktops acquire labels in two ways:
 
-1. The `host_labels` rules defined in the `windows_desktop_service` section of
-   your Teleport configuration file.
-2. Automatic `teleport.dev/` labels applied by Teleport (for desktops discovered
-   via LDAP only)
+1. Via the `host_labels` rules defined in the `windows_desktop_service` section
+   of your Teleport configuration file.
+2. Via LDAP (for desktops discovered via LDAP only)
```

Comment:
I think it would be good here to add a brief (<= one-sentence) note about how LDAP-based discovery works in Teleport. Since this is a conceptual/reference guide, I think we can link to our docs on LDAP-based discovery.